src/ecosystem/Ecosystem.jl

```diff
@@ -207,6 +207,18 @@ function create_coral_struct(bounds::Tuple{Float64,Float64}=(0.9, 1.1))::Nothing
     return
 end
 
+"""
+    colony_areas(colony_diam_means)
+
+Generate colony areas for each size class.
+
+# Arguments
+- `colony_diam_means` : mean colony diameter (in meters)
+
+"""
+function colony_mean_area(colony_diam_means)
```

Comment:
Would `mean_colony_area()` make more sense for readability?

This looks like a parameter naming scheme I use: subject first (i.e., colony) so you don't get a million false matches when doing a quick search.

That said, it matches the naming scheme of `colony_areas()` below, so makes sense we leave it as is.

Reply:
Yeah I made it similar to colony_areas because they're related, but happy to change

Reply:
It's consistent, so happy to leave as is.